Xamarin.Forms.Core/SwipeItems.cs

```diff
@@ -15,6 +16,18 @@ public SwipeItems()
 			_internal.CollectionChanged += OnSwipeItemsChanged;
 		}
 
+		public SwipeItems(IList<ISwipeItem> swipeItems)
```

Comment:
```suggestion
		public SwipeItems(IEnumerable<ISwipeItem> swipeItems)
```

Should this be IEnumerable?

The constructor on ObservableCollection is already going to box it to an IEnumerable and doesn't have an IList constructor
```C#
public ObservableCollection(IEnumerable<T> collection);
```

Reply:
```suggestion
		public SwipeItems(IEnumerable<ISwipeItem> swipeItems)
```

Should this be IEnumerable?

The constructor on ObservableCollection is already going to box it to an IEnumerable and doesn't have an IList constructor
```C#
public ObservableCollection(IEnumerable<T> collection);
```